src/utils/validators/graphqlQueryValidators.ts

```diff
@@ -20,7 +20,7 @@ const resourcePerDateRegex = new RegExp(
 );
 
 const ethereumWalletAddressRegex = /^0x[a-fA-F0-9]{40}$/;
-const solanaWalletAddressRegex = /^[A-Za-z0-9]{43,44}$/;
+const solanaWalletAddressRegex = /^(?:[A-Za-z0-9]{32}|[A-Za-z0-9]{43,44})$/;
```

Comment:
Solana ProgramIds are 43 to 44 character long. There is only one exception, system program id, which is 11111111111111111111111111111111 
So this must work:
const programIdRegex = /^(11111111111111111111111111111111|[1-9A-HJ-NP-Za-km-z]{43,44})$/;